app/processors/base_processor.rb

```diff
@@ -255,10 +264,8 @@ def download_file(uri)
   end
 
   def file_schemes
-    @_file_schemes ||= begin
-      s = ['s3', 'ia', 'ftp', 'http', 'https']
-      s << 'file' if SystemInformation.env != 'production'
-      s
+    @_file_schemes ||= ['s3', 'ia', 'ftp', 'http', 'https'].tap  do |fs|
+      fs << 'file' if SystemInformation.env != 'production'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.